src/utils.py

```diff
@@ -6,62 +6,45 @@
 """
 import logging
 import subprocess
+from pathlib import Path
 
 logging.getLogger().setLevel(logging.INFO)
 
 
 class DataHandler:
-    def __init__(self, data_path, dataset_name):
+    def __init__(self, data_path):
         self.data_path = data_path
-        self.dataset_name = dataset_name
         logging.info(f"Data Handler Class Created - {data_path}")
-        if dataset_name == "Indosum":
-            self.download_indosum()
 
-    def download_indosum(self):
+    def downloader(self, download_type):
         """
-        Downaloads Indosum in data_path/indosum.
+        Downaloads download_type in data_path/download_type.
         """
-        logging.info("Downloading Indosum")
-        if not self.check_data_exists(self.dataset_name):
-            (self.data_path / self.dataset_name).mkdir(parents=True, exist_ok=True)
-        else:
-            logging.info("Folder already exists")
-        if not (self.data_path / self.dataset_name / "indosum.tar.gz").exists():
+        logging.info(f"Downloading {download_type}")
```

Comment:
```suggestion
        download_logger.info(f"Downloading {download_type}")
```

Reply:
And similar changes everywhere else. We're supposed to use an object of the logging class, not the `classmethod`